File: backend/src/main/java/com/backend/domain/repository/dto/response/github/CommitResponse.java
package com.backend.domain.repository.dto.response.github;

// commits 응답용 DTO
public record CommitResponse(
        CommitDetails commit
) {
    public record CommitDetails(
            String message,
            AuthorDetails author
    ) {}

    public record AuthorDetails(
            String date
    ) {}
}

File: backend/src/main/java/com/backend/domain/repository/dto/response/github/CommitResponse.kt
package com.backend.domain.repository.dto.response.github

// commits 응답용 DTO
data class CommitResponse(
    val commit: CommitDetails?
) {
    data class CommitDetails(
        val message: String?,
        val author: AuthorDetails?
    )

    data class AuthorDetails(
        val date: String?
    )
}
